boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -60,34 +59,159 @@ currently supports the following hardware features:
    * - GPIO
      - on-chip
      - gpio
+   * - ADC
+     - on-chip
+     - adc
+   * - I2C
+     - on-chip
+     - i2c
+   * - KSCAN
+     - on-chip
+     - kscan
+
+
+Other hardware features are not currently supported by Zephyr (at the moment)
```

Comment:
Nit - no need to include "(at the moment)".  Please add a closing period "."

Reply:
Sorry for delay. Fixed it.